assets/src/util/dateTime.ts

```diff
@@ -104,3 +104,21 @@ export const timeAgoLabel = (
 
   return diffHours >= 1 ? `${diffHours} hours ago` : `${diffMinutes} min ago`
 }
+
+const second = 1000
+const minute = second * 60
+const hour = minute * 60
+
+export const timeAgoLabelForDates = (start: Date, end: Date) => {
```

Comment:
What's the purpose / difference between this and `timeAgoLabel`? The input / output seems spiritually the same (even if they take different input types)

Reply:
The main difference is the addition of "Just now" but I made a separate function for "development speed" sake, I'll see about condensing these!

Reply:
how's this? https://github.com/mbta/skate/compare/08685fb499b5cbfb8a076b3d605cd401047bfb5d..700b88734064884c7f03bdc1dbb6a3392794a6e6

Reply:
I am a little surprised by the snapshot change, so I'm gonna review that tomorrow (grr, snapshots >.>)

Reply:
Yeah, looks good! Just... grr snapshots

Reply:
Okay fixed the snapshots! It was the "seconds" value 😅